script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -563,93 +611,95 @@ A cold wasteland where everything is dead.
 [sha:6fff90333dc79f996d6fb6cabe292f907bc39fc6]{
 -- Page 438, Offset 32931.
 -- ……物事の“死”が視えるという事は、すべてがあやふやで脆い世界に投げ込まれる事だ。
-...To see the 'death' of things is to be thrown into a world where everything is vague and brittle.
+...To see the 'death' of things is to be thrown into a world where everything is brittle and uncertain.
 }
 [sha:6cb4d574f6fe8e6650fe23534a014318b35e02a8]{
 -- Page 438, Offset 32932.
 -- 地面は無いに等しく、空は今にも落ちてきそうなほど、弱々しい。
-So frail that the ground is like nothingness, and the skies could fall at any moment.
+A world so frail that the ground is almost insubstantial, and the sky could fall at any moment.
```

Comment:
I see where you're getting "frail" from but I think "fragile" would be more natural